testsuite/features/support/twopence_init.rb

```diff
@@ -306,14 +305,6 @@ def client_public_ip(host)
   node = $node_by_host[host]
   raise "Cannot resolve node for host '#{host}'" if node.nil?
 
-  # For each node that we support we must know which network interface uses (see the case below).
-  # Having the IP as an attribute is something useful for the clients.
-  # Let's not implement it for nodes where we are likely not need this feature (e.g. ctl).
-  not_implemented = [$localhost]
-  not_implemented.each do |it|
-    return 'NOT_IMPLEMENTED' if node == it
-  end
-
```

Comment:
Why remove this now?

Reply:
In the past, we were not using our local machine as a controller to debug and develop our tests, so at any point you need the IP of localhost, but running the test from my IDE, there is a piece of code that fails if we don't have localhost node with a proper ip defined. And actually, I don't see a good reason to don't populate it.